equistore-torch/src/misc.cpp

```diff
@@ -0,0 +1,45 @@
+#include <c10/core/DeviceType.h>
+#include <c10/core/TensorOptions.h>
+#include <c10/util/intrusive_ptr.h>
+#include <sys/_types/_int64_t.h>
+#include <torch/torch.h>
+
+#include <equistore.hpp>
+#include <torch/types.h>
+
+#include "equistore/torch/array.hpp"
+#include "equistore/torch/misc.hpp"
+
+using namespace equistore_torch;
+
+
+eqs_status_t equistore_torch::details::create_torch_array(
```

Comment:
Note for myself: check that we enclose this function in a try-catch block in the C++ API, or do it here

Reply:
Done